compiler-no-html/package.js

```diff
@@ -7,7 +7,7 @@ Package.describe({
 
 Package.registerBuildPlugin({
   name: 'svelte-compiler',
-  use: ['svelte:core@1.2.1'],
+  use: ['svelte:core@1.23.4_1'],
```

Comment:
Just ignore everything I just wrote... Something broke when I was thinking 😝 

Anyways, we should still be using the latest version of the compiler even in this deprecated version, no?